.github/styles/Vocab/Products/accept.txt

```diff
@@ -1,44 +1,81 @@
+721s
 ABIs
 addValidator
 AddValidator
+Agrawal
+aggresive
+Aggresive
+Akanimoh
 Algolia
 alloc
 Alloc
+alloweds
 ANR's
+Ansible
 API's
 APIs
 architecting
+Arweave
+auctionId
 auditable
 autogenerated
 Ava
 Ava Labs
 Avalanche-CLI
+avalanchego
 AvalancheGo
 Avascan
+avax
+Avax
 AVAX
+Berkay
+Binance
+bintools
+Blanco
+blackhole
 blockchain
 Blockchain
 blockchains
 Blockchains
+blockNumber
 blockTimestamp
 bool
+bridgeAdmin
+Brunain
+BUIDLing
 calldata
 CAPTCHA
 ChainBridge
+chainlink
+Chainlink
+chainlist
 ChainSafe
+cid
 CLIs
 codec
 Codec
 coinbase
 composability
 Composability
+Consensys
 config
 Config
+Connor
+Corbellini
 Coreth
 Crabada
+crypto
+cryptocurrencies
+cryptocurrency
+Ctrl
+Daly
+DAO
 dapp
+Dapp
```

Comment:
@dasconnor what is your opinion on dapp/Dapp?

Reply:
Should always be `dapp` unless in title case at the beginning of a sentence.

Reply:
That's what we're doing.

Reply:
Resolved